src/ui/Folding/Folding.ts

```diff
@@ -79,30 +81,44 @@ export class Folding extends Component {
      * Specifies the name of the field on which to do the folding.
      *
      * Specifying a value for this option is required for this component to work.
+     *
+     * Default value is `@foldingcollection`.
      */
-    field: ComponentOptions.buildFieldOption({ required: true }),
-
+    field: ComponentOptions.buildFieldOption({ defaultValue: '@foldingcollection' }),
     /**
      * Specifies the field that determines whether a certain result is a child of another top result.
-     *
      * **Note:**
      * > In the index, the values of the corresponding field must contain alphanumerical characters only. Using a
-     * > `childField` whose values contain non-indexable characters (such as underscores) will make folding fail.
+     * > `child` whose values contain non-indexable characters (such as underscores) will make folding fail.
      *
-     * Default value is `@topparentid`.
+     * Default value is `@foldingchild`.
      */
-    childField: ComponentOptions.buildFieldOption({ defaultValue: '@topparentid' }),
-
+    child: ComponentOptions.buildFieldOption({ defaultValue: '@foldingchild' }),
     /**
      * Specifies the field that determines whether a certain result is a top result containing other child results.
      *
      * **Note:**
      * > In the index, the values of the corresponding field must contain alphanumerical characters only. Using a
-     * > `parentField` whose values contain non-indexable characters (such as underscores) will make folding fail.
+     * > `parent` whose values contain non-indexable characters (such as underscores) will make folding fail.
      *
-     * Default value is `@containsattachment`.
+     * Default value is `@foldingparent`.
      */
-    parentField: ComponentOptions.buildFieldOption({ defaultValue: '@containsattachment' }),
+    parent: ComponentOptions.buildFieldOption({ defaultValue: '@foldingparent' }),
+
+    /**
+     * This option has been deprecated and should be replaced with {@link Folding.options.parent}
+     * @deprecated
+     */
+    childField: ComponentOptions.buildFieldOption({
+      deprecated: 'This option has been deprecated and should be replaced with data-parent'
+    }),
+    /**
+     * This option has been deprecated and should be replaced with {@link Folding.options.child}
```

Comment:
This option is deprecated. Instead, use the {@link Folding.options.child} option.